datacapturing/src/movebis/java/de/cyface/datacapturing/MovebisDataCapturingService.java

```diff
@@ -355,23 +356,17 @@ public void start(@NonNull Vehicle vehicle, @NonNull StartUpFinishedHandler fini
         try {
             super.start(vehicle, finishedHandler);
         } catch (final CorruptedMeasurementException e) {
+            final List<Measurement> corruptedMeasurements = new ArrayList<>();
             final List<Measurement> openMeasurements = this.persistenceLayer.loadMeasurements(MeasurementStatus.OPEN);
-            for (final Measurement measurement : openMeasurements) {
-                Log.w(TAG, "Cleaning and finishing dead open measurement (mid " + measurement.getIdentifier() + ").");
-                this.persistenceLayer.deletePoint3dData(measurement.getIdentifier());
-                try {
-                    this.persistenceLayer.setStatus(measurement.getIdentifier(), MeasurementStatus.FINISHED);
-                } catch (NoSuchMeasurementException e1) {
-                    throw new IllegalStateException(e);
-                }
-            }
-            final List<Measurement> pausedMeasurements = this.persistenceLayer
-                    .loadMeasurements(MeasurementStatus.PAUSED);
-            for (final Measurement measurement : pausedMeasurements) {
-                Log.w(TAG, "Finishing dead paused measurement (mid " + measurement.getIdentifier() + ").");
+            final List<Measurement> pausedMeasurements = this.persistenceLayer.loadMeasurements(MeasurementStatus.PAUSED);
+            corruptedMeasurements.addAll(openMeasurements);
+            corruptedMeasurements.addAll(pausedMeasurements);
+
+            for (final Measurement measurement : corruptedMeasurements) {
+                Log.w(TAG, "Finishing corrupted measurement (mid " + measurement.getIdentifier() + ").");
                 try {
                     this.persistenceLayer.setStatus(measurement.getIdentifier(), MeasurementStatus.FINISHED);
-                } catch (NoSuchMeasurementException e1) {
```

Comment:
You are catching an exception called `e1` but you are rethrowing `e` from above. This is probably not what you intended to do.